_pages/yaml-file-format.md

```diff
@@ -184,6 +187,19 @@ Their names may be changed a YAML file with a `lang` other than `en`.
     Standard structures may have an `extension tags` entry to list *fully compatible* extensions that predated the standard and can be converted to the `standard tag` without any other modification.
     For example, 7.0's `UID` structure is fully compatible with the common 5.5.1 extension identified by tag `_UID`.
 
+-   <table><tbody>
+    <tr><th>Key</th><td><code>fragment</code></td></tr>
+    <tr><th>Type</th><td><code>str</code></td></tr>
+    <tr><th>Required by</th><td>—</td></tr>
+    <tr><th>Allowed by</th><td><code>type: uri</code></td></tr>
+    </tbody></table>
+    
+    When combined with the fragment identifer of a URI,
+    creates a brief name or label to show to users to describe the fragment identifier.
+    It is recommended that applications use this instead of `label` in UI elements.
```

Comment:
```suggestion
A recommended brief name or label for a fragment identifier, to show to users. Labels are user-centric; for programmer-centric explanations of the concept, see `specification`.

Labels are short to fit in forms and other constrained-space UI elements; for more detailed text see `help text`.
```

This is not "instead of" label, it's in addition to label (every entry with a "fragment" also has a "label" too, but the UI values are different... fragment identifier vs structure payload).

I'd also change line 212 from "see `label`" to "see `label` and `fragment`".
And I'd change line 221 from
"A recommended brief name or label to show to users."
to something like
"A recommended brief name or label for a structure payload, to show to users."